gradle.properties

```diff
@@ -1,3 +1,9 @@
+besuVersion=22.10.3
```

Comment:
Could we update this?

Reply:
No unfortunately as newer versions use Java 17. And we blocked by the Azure issue to use Java 17 at the moment.